artiq/coredevice/comm_kernel.py

```diff
@@ -651,6 +652,9 @@ def read_exception_string():
             message = read_exception_string()
             params = [self._read_int64() for _ in range(3)]
 
+            # Add channel name
+            params[1] = str(params[1]) + self.config_channel_name(params[1])
```

Comment:
Stop needlessly mutating variables and even changing their type. This is a footgun. Just create a new variable with the channel name if you need to.

Reply:
> Just create a new variable with the channel name if you need to

Do you mean I can add the var to the message in core device?